packages/eds-core-react/src/components/Chip/Chips.docs.mdx

```diff
@@ -0,0 +1,45 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Chip } from '.'
+
+# Chip
+
+Chips, also known as tags or badges, are advanced badges that represent discrete information.
+
+<Links 
+    figmaUrl="https://www.figma.com/file/0bGXR2sCwMVSDNyyzu5BXrO5/UI%E2%80%94User-Interface?node-id=18618%3A0"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/95e67f-chip/b/95ad56"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Chip/Chip.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+></Links>
+
+<Story id="data-display-chips--introduction"/>
+<PropsTable />
+
+## Usage
+
+Chips allow users to enter information, make selections, filter content, or trigger actions.
+<ul>
+    <li>Chips are <strong>compact</strong> components that represent discrete information.</li>
+    <li>Chips should be <strong>relevant</strong> to the content or task they represent.</li>
+    <li>Chips should make tasks easier to complete, or content easier to sort.</li>
+</ul>
```

Comment:
I like this short and sweet usage helper text! 

One question though. Where do we have guidelines for do's and don't as this is already described on our Storefront docs for chips [here](https://eds.equinor.com/0b0c666ab/p/95e67f-chip/b/49f4b2). If we write information like this here we have to make sure its in sync with whats on our Storefront at any given time, and who will be responsible for keeping this up to date? Any inputs @BeckyBrekke ?